web/src/pages/team/[team_id]/kiosk/[action_id].tsx

```diff
@@ -35,16 +35,16 @@ const actionKioskQuery = gql`
       app {
         id
         is_staging
-        app_metadata(where: { status: { _neq: "verified" } }) {
+        app_metadata(where: { verification_status: { _neq: "verified" } }) {
           name
-          status
+          verification_status
         }
         verified_app_metadata: app_metadata(
-          where: { status: { _eq: "verified" } }
+          where: { verification_status: { _eq: "verified" } }
         ) {
           name
           logo_img_url
-          status
+          verification_status
```

Comment:
fyi and for @andy-t-wang, the app model still has status & AppStatusType may need updates

Reply:
That's the overall app status for the user to activate or deactivate the overall app. Left it there since I figured it made more sense for that to be stored there to prevent things going out of sync